content/messaging/customer-chat-api/v3.2/index.mdx

```diff
@@ -1081,10 +1081,18 @@ It returns [summaries](#chat-summary) of the chats a Customer participated in.
 
 #### Request
 
-| Parameter | Required | Type     | Notes                    |
-| --------- | -------- | -------- | ------------------------ |
-| `offset`  | No       | `number` | Default: 0, maximum: 100 |
-| `limit`   | No       | `number` | Default: 10, maximum: 25 |
+| Parameter | Required | Type     | Notes                     |
+| --------- | -------- | -------- | ------------------------- |
+| `limit`   | No       | `number` | Default: 10, maximum: 25  |
+| `page_id` | No       | `string` |                           |
+
+#### Response
+
+| Parameter          | Data type | Notes                                                                                                           |
+| ------------------ | --------- | --------------------------------------------------------------------------------------------------------------- |
+| `total_chats`      | `number`  | An estimated number. The real number of found chats can slightly differ.                                        |
+| `next_page_id`     | `string`  | In relation to `page_id` specified in the request. Appears in the response only when there's the next page.     |
```

Comment:
Could you explain somewhere how the pagination works for this method? By looking at the information given here I have no idea how it works and, as a developer, I'm not sure how to use this. 

My main doubt is this:
- I perform a `list_chats` request with `limit: 2` while having A, B, C, D, E chats (in that order)
- I get back D, E chats and some `next_page_id`
- chat C is being reactivated
- I perform another `list_chats` request with the previously received `next_page_id`

Which chats do I get in return? 🤔

Reply:
Obviously you will receive A, B chats because in this they method are ordered by last thread creation date. I will put it into docs